plugins/qCompass/ccCompassInfo.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CC_COMPASS_INFO_HEADER
+#define CC_COMPASS_INFO_HEADER
+
+#include <qdialog.h>
+#include <qlabel.h>
+#include <qpushbutton.h>
+#include <qdialogbuttonbox.h>
+#include <qboxlayout.h>
+#include <qtextedit.h>
+
+class ccCompassInfo : public QDialog
+{
+	
+public:
+	explicit ccCompassInfo(QWidget *parent = 0) :
+		QDialog(parent)
+	{
+
+		setFixedWidth(800);
+		setFixedHeight(600);
+
+		//setup GUI components
+		QTextEdit* l = new QTextEdit(this);
+		l->acceptRichText();
+		l->setReadOnly(true);
+
+		QPushButton *okButton = new QPushButton(tr("Ok"));
+		QDialogButtonBox *buttonBox = new QDialogButtonBox(Qt::Horizontal);
+		buttonBox->addButton(okButton, QDialogButtonBox::AcceptRole);
+		connect(buttonBox, SIGNAL(accepted()), this, SLOT(accept())); //done
+		QVBoxLayout *lt = new QVBoxLayout;
+		lt->addWidget(l);
+		lt->addWidget(buttonBox);
+		setLayout(lt);
+
+		//add text
+		l->setText("<!DOCTYPE html>"
```

Comment:
I agree completely - though I'm unsure how to do this such that the file gets included in the compiled binary (avoiding relative path issues etc. on distribution)